enterprise/utils.py

```diff
@@ -143,6 +143,34 @@ def get_idp_choices():
     return None
 
 
+def get_provider_login_url(request, provider_id):
+    """
+    Return the given provider's login URL.
+    """
+    try:
+        from third_party_auth import pipeline
+    except ImportError as exception:
```

Comment:
In what circumcetances we will be getting this Import Error?

Reply:
We are importing this from edx-platform. As enterprise got installed in edx-platform and we are practicing this from start.

Reply:
Would it be a better design to keep the backend in edx-platform and only call the enterprise to get the IDP url? ... bringing the pipeline into enterprise is creating a lot of unnecessary coupling ...

Reply:
@irfanuddinahmad excellent suggestion. FYI we are already doing this kind of [thing](https://github.com/edx/edx-enterprise/blob/e656a52bc58cff879fcebc301bac25a601c639e0/enterprise/tpa_pipeline.py#L23) already. But I have incorporated your feedback. I have created the same method in `edx-platform/openedx/features/enterprise_support/utils.py`.